### ISSUE TITLE
"Up time" return at 0

### ISSUE THREAD

**Issue by hubyhuby** [syncthing/syncthing-android#875]
### Version Information

    App Version: 0.9.8 & 0.9.3
    Syncthing Version: v0.14.25
    Android Version: Android 6.0 / Samsung A36 & Android 4.4 / Sony xperia E3

With two different phones I am experiencing "Up time" that return to "less than 1 min" all the time.
When I go in the "Web interface", It is allmost always  below 15 minutes on both devices. The max I managed was 1H.

My understanding of "Up time", is the duration of the syncthing service running.
Therefore I believe it is a problem for both battery life and efficiency in general.

To reproduce this behaviour I simply install Syncthing 0.9.8 and do nothing else.
Then come back every 15 minutes and see uptime below 15 minutes in web interface..
 
Often the web interface would also take very long or crash at 2nd or 3rd check and run "waiting web interface" for  15minutes or until it crashes.  If I am very patient (15min) the web interface appears and Uptime is back to "below 1 min". I can imagine, this is because we are trying to connect to a dead service. On the android 4.4 it is sometime more expressif and gives a page saying there is no one at 127.0.0.1

Notes:
I am using an unmodified syncthing from the playstore

EDIT:
I have one folder synchronised from a previous installation. An did not re echange the keys between devices, after the last instalaltion.

**Comment by wweich** [comment]
https://github.com/syncthing/syncthing-android/wiki/Frequently-Asked-Questions#why-is-syncthing-stopped-after-some-time

**Comment by hubyhuby** [comment]
Dear @wweich , Thank you for your answer. This explains a part of the issue.
Also this pop s up other issues:
1)
I have installed several times the application and each time I take care to push the pop up button "do not optimise for this application".  Still for an unknown reason Syncting stays in Automatique optimisation on my galaxy A3 2016. I have double checked right now , un install/re install syncthing + "do not optimize batterie" =YES + "Optimisation of battery"= Stop for syncthing" pop ups. Still I have to go manually in battery menu.
Now I could change it manually has you recommended and it seems to work. Also why doesn t the pop up doest it ?


2)In the battery menu in the "deactivate" application section, I see Whatsapp and FB Messenger arrived automatically I beleive... 
To check, I just did the same as with SYncthing "un install"/reinstall " FB messenger and it AUTOMATICALLY arrives in the section "deactivate" of the android battery manager. WITHOUT any pop up talking about battery optimisation.
